charts/camunda-platform-alpha/templates/camunda/_helpers.tpl

```diff
@@ -539,7 +550,7 @@ Zeebe templates.
   {{- if .Values.zeebe.enabled -}}
     {{-
       printf "http://%s:%v%s"
-        (include "zeebe.fullname.gateway" .)
+        (include "zeebe.names.gateway" .)
```

Comment:
FYI: I changed this so that `.Values.global.zeebeClusterName` is considered when creating the URL.

Reply:
I don't see any problems with this but thanks for pointing it out.